### PR TITLE
Feature/credential-mint

### PR DESCRIPTION
**Pull request by NunoMartins21**
# Feature/credential-mint

## Type of PR

- [x] - feature

## Description

Allow users to mint subsidized credential NFTs if they complete a gate/credential on the platform

### CODE REVIEW

File: apps/website/hooks/use-mint.tsx [hunk 1]
```diff
@@ -239,37 +262,61 @@ export function useBiconomyMint(
...
+                : 'https://rinkeby.etherscan.io') +
+              '/tx/' +
+              tx,
           };
         } else {
           console.log('Sending normal transaction');
 
           const tx = await contract.mint(address.address, token_uri);
 
           console.log(
```
Comment (rafagomes) [inline apps/website/hooks/use-mint.tsx:290]: Is this console.log really necessary?

**Review by rafagomes — APPROVED** [review]
Only the console.log, but is not blocking

File: apps/website/hooks/use-mint.tsx (hunk 1 above)
Comment (NunoMartins21) [reply apps/website/hooks/use-mint.tsx:290]: It's never necessary haha. We were just checking for normal vs subsidized transactions, but I'll remove it!

File: apps/website/components/organisms/mint-modal/mint-select.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          ))}
+        </List>
+      </DialogContent>
+      <DialogActions sx={{ paddingBottom: '24px' }}>
+        <Button onClick={onCancel} variant="outlined" size="small" fullWidth>
+          {t('common:actions.cancel')}
+        </Button>
+      </DialogActions>
+
+      {/* {error && <div>{error.message}</div>} */}
```
Comment (fbenfraj) [inline apps/website/components/organisms/mint-modal/mint-select.tsx:94]: Remove comment

File: apps/website/components/templates/profile/tabs/OverviewTab.tsx [hunk 3]
```diff
@@ -40,25 +40,22 @@ export function OverviewTab({ user }: Props) {
...
+      section: <ReceivedTab />,
+    },
+    {
+      key: 'activity',
+      label: t('Given'),
+    },
+    {
+      key: 'bookmarked',
+      label: t('Bookmarked'),
+      //section: <ActivityTab />,
```
Comment (fbenfraj) [inline apps/website/components/templates/profile/tabs/OverviewTab.tsx:56]: Remove ?

File: apps/website/providers/biconomy/context.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+type Context = {
+  mint: () => Promise<MintResponse>;
+  mintCredential: (
+    credential: PartialDeep<Credentials>
+  ) => Promise<MintResponse>;
+  mintStatus: MintStatus;
+};
+
+// export const BiconomyContext = createContext<Context>({
```
Comment (fbenfraj) [inline apps/website/providers/biconomy/context.ts:23]: Remove

**Review by fbenfraj — CHANGES_REQUESTED** [review]

File: apps/website/components/molecules/mint-card/screen/default-mint-screen.tsx [hunk 5]
```diff
@@ -132,22 +144,42 @@ export const DefaultMintScreen = ({
...
-              <OpenInNewIcon fontSize="medium" color="disabled" />
-            </ListItemIcon>
-            Open on Ceramic
-          </MenuItem> */}
+          {details.credential.status == 'minted' && (
+            <MenuItem>
+              <a
+                href={details.credential.transaction_url}
+                target="_blank"
+                rel="noreferrer"
```
Comment (rafagomes) [inline apps/website/components/molecules/mint-card/screen/default-mint-screen.tsx:152]: just to let you know...the new application on the browsers, add this automatically. Is not necessary anymore...

File: apps/website/providers/biconomy/context.ts (hunk 4 above)
Comment (NunoMartins21) [reply apps/website/providers/biconomy/context.ts:23]: Fixed

File: apps/website/components/templates/profile/tabs/OverviewTab.tsx (hunk 3 above)
Comment (NunoMartins21) [reply apps/website/components/templates/profile/tabs/OverviewTab.tsx:56]: Fixed

File: apps/website/components/organisms/mint-modal/mint-select.tsx (hunk 2 above)
Comment (NunoMartins21) [reply apps/website/components/organisms/mint-modal/mint-select.tsx:94]: Fixed

**Pull request merged**